app/src/main/java/org/projectbuendia/client/ui/OdkActivityLauncher.java

```diff
@@ -102,6 +132,11 @@ public static void fetchAndShowXform(
         final int requestCode,
         @Nullable final org.odk.collect.android.model.Patient patient,
         @Nullable final Preset fields) {
+        LOG.i("Trying to fetch it from cache.");
```

Comment:
Vinicius - I think this behaviour is better as it will make the opening of forms more responsive and at the moment we saw in field testing that delay of a few seconds on opening a form - just when you want to add data - is annoying.

Please update the PR description to reflect what you are actually doing.

Now - we do need a way to invalidate the cache and force retrieval of forms from the server. How might we do that? Maybe add a "Clear forms cache" option in our App Settings.

Later, we'll need to think about how that will work with deployment of new forms from HQ to field projects.